gcloud/pubsub/subscription.py

```diff
@@ -305,8 +305,10 @@ def set_iam_policy(self, policy, client=None):
         client = self._require_client(client)
         path = '%s:setIamPolicy' % (self.path,)
         resource = policy.to_api_repr()
+        # 'set_iam_policy' API requires an extra wrapper. :(
+        wrapped = {'policy': resource}
```

Comment:
No, the API is.  The docs actually *say* what the API requires, but the behavior is so silly I mentally elided it away until actually exercising the API in a system test.